abacatepay/_models.py

```diff
@@ -1,52 +1,69 @@
-from typing import List, Optional, Union, Literal
+from typing import List, Union
 from ._constants import BILLING_STATUS, BILLING_METHODS, BILLING_KINDS
 from pydantic import BaseModel
 
+
 class Product(BaseModel):
-  externalId: str
-  name: str
-  quantity: int
-  price: int
-  description: str
+    externalId: str
+    name: str
+    quantity: int
+    price: int
+    description: str
+
 
 class Customer(BaseModel):
     taxId: str
     name: str
     email: str
     cellphone: str
 
-class ICostumer:
-    def __init__(self, data:dict):
+
+class Costumer:
```

Comment:
two classes with the same name, it will cause type and import error

Reply:
Not necessarily
It should work if we use relative imports. Anyways, I'd rather change the `ICostumer` to `Costumer_response` instead of just `Costumer`. Same for `IBilling`